### DISCUSSION TITLE
Meta Open Graph - WikiJS site preview - Rendering image preview

### DISCUSSION THREAD

**Discussion by adventurist** [requarks/wiki#4014] [Help / Questions]
I have looked through the documentation, admin settings, github issues and this discussion forum, but I still haven't figured out how I can set the meta image tag for pages (or even for the home page)

for example:

```<meta property="og:image" content=“https://site.com/image.jpg”>```

I have even tried injecting JS or using HTML injection into Head tag, but this doesn't seem to solve the problem.

How can I configure my WikiJS site so that preview images are generated when sharing in applications or on social media?

**Comment by adventurist** [comment]
Bump

**Comment by SinTan1729** [comment]
This won't work for everyone, but if you use Caddy to serve your WikiJS instance, I wrote a plugin to do just this.
https://github.com/SinTan1729/caddy-wikijs-metatags
